tests/trainer/test_lr_finder.py

```diff
@@ -262,3 +264,12 @@ def test_suggestion_with_non_finite_values(tmpdir):
 
     assert before_lr == after_lr, \
         'Learning rate was altered because of non-finite loss values'
+
+
+def test_lr_finder_fails_fast_on_bad_config(tmpdir):
+    """ Test that tune fails if the model does not have a lr BEFORE running lr find """
+    # note: this did not raise an exception before #5648 because lr_find is skipped
+    # during fast_dev_run and the lr attribute check was done after lr_find
+    trainer = Trainer(default_root_dir=tmpdir, fast_dev_run=True, auto_lr_find=True)
```

Comment:
with `fast_dev_run=True`, lr_find is disabled, how is it generating this error here?

Reply:
should tuner be enabled with fast_dev_run? cc @SkafteNicki

Reply:
the intention is that tuner should be disabled during fast_dev_run

Reply:
Agree with @SkafteNicki 

The point of this test is to ensure that the lr attribute check is done before running lr_find

Reply:
but for batch_size_scaling, its opposite. It checks only if it's going to run it but here it first checks and then decide whether to run.

Reply:
ok I see, there are two different method calls for lr_find but only one for batch_size_caling:

Reply:
can we have a check for `fast_dev_run` inside `_run_lr_finder_internally` too? just to have the same behavior in both?

Reply:
i think that is a great idea @rohitgr7 !

Reply:
cool, let's do in a follow up PR. I have a better suggestion for this.